core/src/main/java/io/paradoxical/cassieq/workers/SimpleRepairWorkerManager.java

```diff
@@ -21,15 +22,28 @@ public SimpleRepairWorkerManager(Provider<QueueRepository> queueRepositoryProvid
         this.repairWorkerFactory = repairWorkerFactory;
     }
 
-    @Override public void start() {
-        activeRepairWorkers = queueRepositoryProvider.get().getQueues().stream().map(repairWorkerFactory::forQueue).collect(toList());
+    @Override public synchronized void start() {
```

Comment:
we need to chat about this, because not sure the repair worker scales if the number of queues is in the thousands. :/